middleware/checkJwt.js

```diff
@@ -1,5 +1,11 @@
 const jwt = require('express-jwt');
 const jwksRsa = require('jwks-rsa');
+const dotenv = require('dotenv');
```

Comment:
Potentially redundant: dotenv.config() is called in index.js

Reply:
Done. 👍